app/controllers/org_admin/questions_controller.rb

```diff
@@ -0,0 +1,213 @@
+module OrgAdmin
+  class QuestionsController < ApplicationController
+    include Versionable
+  
+    respond_to :html
+    after_action :verify_authorized
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:section_id]/questions
+    def index
+      authorize Question.new
+      section = Section.includes(:questions, phase: :template).find(params[:section_id])
```

Comment:
do we need to pull the section, the phase and the template to display a bunch of questions?